cpp/CMakeLists.txt

```diff
@@ -798,13 +800,16 @@ target_compile_definitions(cudf PRIVATE "RMM_LOGGING_LEVEL=LIBCUDF_LOGGING_LEVEL
 # Define spdlog level
 target_compile_definitions(cudf PUBLIC "SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${LIBCUDF_LOGGING_LEVEL}")
 
+# Enable pause functions in thread_pool implementation
+target_compile_definitions(cudf PRIVATE "BS_THREAD_POOL_ENABLE_PAUSE=1")
```

Comment:
We should move this off `cudf` and onto the `BS_thread_pool` target in `get_thread_pool.cmake` as an INTERFACE compile definition.

```cmake
target_compile_definitions(BS_thread_pool INTERFACE "BS_THREAD_POOL_ENABLE_PAUSE=1")
```

Reply:
Thanks, done.